### ISSUE TITLE
Please add AF Microadjustment

### ISSUE THREAD

**Issue by GoogleCodeExporter** [400plus/400plus#106]
```
It would be nice to have autofocus microadjustment like the one available on 
50D firmware. 
Could be very useful for Sigma branded lens, well knows for front/back focus 
issues. 

http://www.northlight-images.co.uk/article_pages/cameras/1ds3_af_micoadjustment.
html
```

Original issue reported on code.google.com by `effe...@gmail.com` on 6 Jul 2011 at 9:00
- Blocking: #309
- Blocked on: #117
